storybook/theme.js

```diff
@@ -6,6 +6,36 @@ import { create } from '@storybook/theming/create';
 export default create( {
 	base: 'light',
 	brandTitle: 'WordPress Components',
-	brandImage:
-		'https://s.w.org/style/images/about/WordPress-logotype-wmark.png',
+	brandImage: './wp-org-logo.png',
+
+	// Typography
+	fontBase: '"Inter", sans-serif',
+	fontCode: 'monospace',
+
+	//
+	colorPrimary: '#3858E9',
+	colorSecondary: '#3858E9',
```

Comment:
How would the secondary color affect the UI? Could we use a different color, or are we ok with the same as primary?

Reply:
I haven't been able to figure out what that secondary color affects, so I changed it to our primary, just in case. There are still a few tints of blue and orange in some places that this theme doesn't contemplate, I think we'll change those in the next update.

Reply:
As an experiment, I set `colorPrimary` to green and `colorSecondary` to red — the following screenshots should help you understand what parts of the UI are affected by each variable (it looks like `colorPrimary` doesn't basically make any differences?)


![Screenshot 2024-10-01 at 23 25 49](https://github.com/user-attachments/assets/07e1fedc-cd90-4f64-8263-9b248f881ce3)